javaHapiValidatorLambda/serverless.yml

```diff
@@ -0,0 +1,35 @@
+#
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+service: fhir-service-validator
+
+provider:
+  name: aws
+  region: us-west-2
+  stage: dev
+  runtime: java11
```

Comment:
Could you add x-ray tracing too: https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/serverless.yaml#L56

Will be helpful when we do a load test